### PR TITLE
Support plone.app.multilingual 

### PR DESCRIPTION
**Pull request by bierik**
Closes https://github.com/4teamwork/endk.web/issues/2

Requires https://github.com/4teamwork/plonetheme.blueberry/pull/84 which has already been merged.

bildschirmfoto 2016-07-19 um 11 17 30
bildschirmfoto 2016-07-19 um 12 10 00

### CODE REVIEW

**Comment by bierik** [comment]
@jone updated

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,13 +5,15 @@ Changelog
...
+- Pevent scrolling when navigation is open
+  [Kevin Bieri]
 
 
 1.0.1 (2016-07-18)
 ------------------
 
-- Pevent scrolling when navigation is open
+=======
+- Add support for plone.app.multilingual
```
Comment (jone) [inline docs/HISTORY.txt]: looks strange; rebase error?

**Comment by bierik** [comment]
@jone updated

File: ftw/mobile/buttons.py [hunk 2]
```diff
@@ -99,3 +102,31 @@ def endpoint(self):
...
+        return current_language
+
+    def position(self):
+        return 300
+
+    def data(self):
+        result = OrderedDict()
+        portal_languages = self.context.portal_languages
+
+        langs = portal_languages.getAvailableLanguages()
```
Comment (jone) [inline ftw/mobile/buttons.py]: I'm not sure if this is the "plone.app.multilingual" approach and I have no idea how to do this correctly.
I assume it should not be a "switch plone GUI language" switch but a "switch to a different language version of this context" switch.

@maethu can you give your input how this should be done?  I have no idea how this should work.

File: ftw/mobile/buttons.py [hunk 3]
```diff
@@ -99,3 +102,33 @@ def endpoint(self):
...
+        current_language = portal_state.language()
+        return current_language
+
+    def position(self):
+        return 300
+
+    def data(self):
+        portal_languages = self.context.portal_languages
+
+        langs = portal_languages.getAvailableLanguages()
```
Comment (jone) [inline ftw/mobile/buttons.py]: I'm not sure if this is the "plone.app.multilingual" approach and I have no idea how to do this correctly.
I assume it should not be a "switch plone GUI language" switch but a "switch to a different language version of this context" switch.

@maethu can you give your input how this should be done?  I have no idea how this should work.

Comment (mbaechtold) [reply ftw/mobile/buttons.py]: @jone, I have pushed a commit onto this branch which might be the solution to your question. Could you please review it?

**Comment by bierik** [comment]
@jone updated please have a look

**Comment by jone** [comment]
@maethu please do the review as you have more knowledge about p.a.multilingual 😉

**Comment by mbaechtold** [comment]
I had to rebase onto `master` due to merge conflicts. I took the opportunity to squash some commits since I had to force push anyway.

**Comment by raphael-s** [comment]
@bierik -> https://github.com/4teamwork/kwl.web/issues/4

**Pull request merged**